src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -498,7 +498,1246 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== How to install ProActive using Kubernetes
 
+ProActive could be easily deployed and started in a Kubernetes cluster. For this purpose, we provide a ready-to-use Kubernetes configuration files that enable you running ProActive Workflows and Scheduling as a set of Pods.
+It starts the following Pods:
+
+*  ProActive Server Pod
+*  ProActive Node Pod
+*  ProActive Database Pod (PostgreSQL or MySQL). The Database Pod is not started when using an embedded HSQL Database.
+
+The Kubernetes YAML configuration pulls the ProActive server, Node, and Database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
+
+To be able to pull images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^], a ProActive enterprise licence and access credentials are required. Please contact us at contact@activeeon.com to request access.
+
+Here are the installation steps you need to follow in order to have ProActive running in a Kubernetes cluster:
+
+====== Set up ProActive Pods using embedded HSQL Database
+The following YAML Kubernetes configuration could be used to run the ProActive server Pod with an embedded HSQL Database.
+It will start up to two Pods, one for the server and one for the Nodes.
+
+[source]
+----
+apiVersion: v1
+data:
+  DB_CATALOG_PASS: SjSWUF72s?C*eWhB
```

Comment:
you should mention which variables in this section can be/should be edited.

As the file is public in the documentation, a customer you should edit passwords 

And I see many parameters below that should be modified, you could provide a table with
the name of the parameter, its description (and whether the parameter should be left as is or modified), the default value